### PR TITLE
FIX: Support message throwing multiple exceptions

### LINKED ISSUE #7: Multiple exceptions in `throws` declaration are marked as errors
Example:
```
protocol Painter {
 
  void draw() throws MyError, AnotherError;
//                          ^---- this comma is highlighted as an error 
}
```

screen shot 2017-09-29 at 14 42 06

### PR DESCRIPTION
**Pull request by shtratos**
Fixes #7

I've fixed the grammar, regenerated the parser and bumped patch version.

Please have a look.

### CODE REVIEW

File: src/claims/bold/intellij/avro/idl/AvroIdl.bnf [hunk 1]
```diff
@@ -95,7 +95,7 @@ enum_value ::= IDENTIFIER
 fixed_decl ::= annotation* 'fixed' IDENTIFIER '(' INT_LITERAL ')' ';' { pin = 2 }
 
 message_decl ::= type IDENTIFIER '(' arguments ')' message_attributes* ';' { pin = 3 }
-private message_attributes ::= 'throws' IDENTIFIER | 'oneway' { pin = 1}
+private message_attributes ::= ( 'throws' IDENTIFIER (',' IDENTIFIER)* ) | 'oneway' { pin = 1}
```
Comment (AbigailBuccaneer) [inline src/claims/bold/intellij/avro/idl/AvroIdl.bnf:98]: Does explicitly grouping the left-hand of the pipe in `()` affect the `pin` attribute? I'm rusty with the inner workings of the grammar.

**Review by AbigailBuccaneer — COMMENTED** [review]
Thanks for your contribution! LGTM other than the one question below, I'll merge once it's answered (assuming the answer is 'no').

File: src/claims/bold/intellij/avro/idl/AvroIdl.bnf (hunk 1 above)
Comment (shtratos) [reply src/claims/bold/intellij/avro/idl/AvroIdl.bnf:98]: I'm not sure. The parser behaves as if `throws` or `oneway` are pinned, so I guess it works. 
I tried changing it and didn't see any changes in the generated parser code.

I can split this rule in two if you think it helps:
```
message_decl ::= type IDENTIFIER '(' arguments ')' (throws_message_attribute | oneway_message_attribute)* ';' { pin = 3 }
private throws_message_attribute ::= 'throws' IDENTIFIER (',' IDENTIFIER)* { pin = 1 }
private oneway_message_attribute ::= 'oneway'
```

**Comment by shtratos** [comment]
Thanks for merging this. Could you please release a new version of plugin, so I can tell my team to update?

**Comment by AbigailBuccaneer** [comment]
"This update is being reviewed by JetBrains, please expect it to be approved in two days after upload"

In the mean time you can download the plugin manually from https://github.com/AbigailBuccaneer/intellij-avro/releases/latest.

**Comment by shtratos** [comment]
Thank you for your help! 🙇

**Comment by AbigailBuccaneer** [comment]
No worries, thank you for your work! :bowing_woman:

The plugin is now available from the JetBrains plugin repository (https://plugins.jetbrains.com/plugin/7971-apache-avro-support), so if you've installed it from file then it's probably best to remove it and install it from the repository. (Unless you're going to make more contributions, which will always be welcome! :innocent:)

**Pull request merged**